### PR TITLE
Ignore development files on production env

### LINKED ISSUE #372: Composer create project script ignore development files on production
### Bug Report

|    Q        |   A
|------------ | ------
| Version(s)  | 5.0.0

#### Summary

Composer create project script ignore development files on production

#### Expected behavior

Ignores the copy of development files on production environments

### PR DESCRIPTION
**Pull request by marioradu05**

### CODE REVIEW

File: bin/composer-post-install-script.php [hunk 1]
```diff
@@ -2,33 +2,50 @@
...
 // phpcs:disable PSR1.Files.SideEffects.FoundWithSymbols
 
 function copyFile(array $file): void
 {
     if (is_readable($file['destination'])) {
         echo "File {$file['destination']} already exists." . PHP_EOL;
     } else {
-        if (! copy($file['source'], $file['destination'])) {
-            echo "Cannot copy {$file['source']} file to {$file['destination']}" . PHP_EOL;
+        if (! in_array(getEnvironment(), $file['environment'])) {
```
Comment (arhimede) [inline bin/composer-post-install-script.php]: The logic chain here can be improved.
First check environment , line 15.
If env is ok, then check if the file exists, then try to copy the file.
 Do not check first if is readable the destination

Comment (marioradu05) [reply bin/composer-post-install-script.php]: @arhimede I've refactored the logic behind the `copyFile` function as requested.

**Review by arhimede — COMMENTED** [review]
please copy the file 
cors.local.php.dist  to cors.local.php 
too 

this is only for API

File: composer.json [hunk 2]
```diff
@@ -103,7 +103,7 @@
         }
     },
     "scripts": {
-        "post-install-cmd": [
```
Comment (arhimede) [inline composer.json:106]: remove this part , according to this issue 
[refactor composer.json - post-create-project-cmd](https://github.com/dotkernel/frontend/issues/531)

**Review by arhimede — APPROVED** [review]

**Pull request merged**